src/components/PrimaryNav/_primary-nav.scss

```diff
@@ -91,6 +84,30 @@
     }
   }
 
+  .p-primary-nav__bottom {
+    border-top: 1px solid #666;
+    margin-top: 0;
+    padding-top: 0.75rem;
+
+    .p-list {
+      &__item {
+        color: #fff;
+        display: flex;
+
+        .p-label--new {
+          margin-left: auto;
+          margin-right: 1rem;
+          margin-top: 0.5rem;
+        }
+
+        .version {
```

Comment:
The versions string seems a bit to prominent, maybe try a smaller font with a grey?

<img width="238" alt="Screen Shot 2020-01-22 at 11 10 05 AM" src="https://user-images.githubusercontent.com/532033/72916521-cb1d9380-3d07-11ea-9ed1-6ba0a1676cd9.png">